Jenkinsfile

```diff
@@ -47,9 +47,19 @@ pipeline {
               sh "cp \$ethereum_password configs/eth/ethereum_password_local.properties"
             }
             sh "./gradlew dependencies"
+            sh "./gradlew dokka"
```

Comment:
dokka artifact is an archive inside another archive. is it possible to make just a single layer?